README.md

```diff
@@ -1,32 +1,22 @@
 Caddy Module: http.handlers.cache
 ================================
 
-**⚠️ Work-in-progress**
-
-This is a distributed HTTP cache module for Caddy.
+This is a distributed HTTP cache module for Caddy based on [Souin](https://github.com/darkweak/souin) cache.
```

Comment:
Perhaps something like this? I think it makes the difference between the two clear.

I dropped the mention of "distributed" feature since that's mentioned below with other features of interest like RFC 7234 compliance.

It's not clear what the default is though, since you have both Badger (non-distributed, but can persist cache to disk?) and Olric (distributed and in-memory only), config docs can cover that or be reviewed on making it simpler (eg a `distributed` boolean instead of user needing to care about Badger or Olric providers and their configs, if they do they can change to Souin plugin).

It should be clear that distributed cache lacks on-disk persistence though (_perhaps that's not a feature `cache-handler` needs to accommodate for though?_).

```suggestion
The official Caddy HTTP cache module. This module embraces the Caddy ethos of keeping configuration simple with smart defaults out of the box!

Under the hood `cache-handler` uses the excellent HTTP caching solution [Souin](https://github.com/darkweak/souin). For advanced features like a REST API for purging cache entries or Surrogate Keys (aka YKey in Varnish), the [Souin Caddy module](https://github.com/darkweak/souin/tree/master/plugins/caddy) can be used directly when needed.
```

Reply:
The single `distributed` boolean option was an option but it was dropped to let the user decide to configure Olric by a configuration file or providing directly the config inside the Caddyfile to set max memory and other Olric options.

Reply:
I'm not sure about the `excellent` mention 😄

Reply:
> For advanced features like a REST API for purging cache entries or Surrogate Keys (aka YKey in Varnish), the [Souin Caddy module](https://github.com/darkweak/souin/tree/master/plugins/caddy) can be used directly when needed.
In fact the REST API is available inside the `cache-handler` but it's not enabled by default due to security reasons. And the `ykey` support may be possible (IMHO it should stay in both modules) but it won't be referenced in the `cache-handler` documentation.